### ISSUE TITLE
CAL bad module configuration format

### ISSUE THREAD

**Issue by oberon2007** [manjaro/release-plan#132]
CAL is complaining about umount.conf format
```
void Calamares::Module::loadConfigurationFile(const QString&) bad module configuration format "/usr/share/calamares/modules/umount.conf"
```
Anything that should/needs to be done about that?

**Comment by oberon2007** [comment]
Seems we are using the default config file there...

**Comment by oberon2007** [comment]
https://github.com/calamares/calamares/issues/707

**Comment by adriaandegroot** [comment]
The issue is that the sample umount.conf is totally empty: contains only comments and blank lines. That's apparently not valid. Adding, say, this line to the conf file gets rid of the warning:

```
totallyBogus: 1
```

**Comment by oberon2007** [comment]
awesome! 😉

**Comment by philmmanjaro** [comment]
I already fixed that in our **v17.02** preview releases with:

```
---
srcLog: "/root/.cache/Calamares/Calamares/Calamares.log"
destLog: "/var/log/Calamares.log"
```

@adriaandegroot: maybe the default config file should then default to those settings :smile:
